### DISCUSSION TITLE
Step CI to use liquidless-faker with @faker-js/faker version 8.X.X

### DISCUSSION THREAD

**Discussion by BapAtRb** [stepci/stepci#205] [General]
Hey! 👋
I was not sure where to post this question, hence forgive me if this is not the correct place.
While working on fake data for our APIs, I have noticed that the faker in Step CI points at version 7 of @faker-js/faker. Could it be rebuilt to point at the newer version?
All the links in other GitHub repos are always redirecting to the instructions of newest version (https://fakerjs.dev). Not all of the APIs described in newest version work with current implementation of Step CI. This introduces a bit of confusion and requires a bit of digging between different repos to find that liquidless-faker depends on @faker-js/faker version 7.6.0 and then the documentation for this version of faker can be found under https://v7.fakerjs.dev.
Thanks! 🙏

**Comment by mishushakov** [comment]
This shouldn't be a problem at all. There was no faker.js v8 when the plugin was being written. Can you open a pull request to update the dependency?

**Reply by BapAtRb** [reply]
Of course! Although, I will need some guidance on what files should be merged via the pull request because I am not a developer, so this will be new to me.

**Reply by mishushakov** [reply]
Then no worries, I finished it for you

**Reply by BapAtRb** [reply]
Thank you very much!

**Comment by mishushakov** [comment]
This is now updated in in liquidless-faker@1.1.1

**Reply by BapAtRb** [reply]
Can it be assumed that next version of Step CI will run on liquidless-faker@1.1.1?

**Reply by mishushakov** [reply]
The current version on NPM already does (needs a reinstall though to upgrade the packages)

**Reply by BapAtRb** [reply]
Right, thank you! 😅
